packages/service-client/odsp-client/api-report/odsp-client.api.md

```diff
@@ -0,0 +1,83 @@
+## API Report File for "@fluidframework/odsp-client"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ContainerSchema } from '@fluidframework/fluid-static';
+import { IClient } from '@fluidframework/protocol-definitions';
+import { IConfigProviderBase } from '@fluidframework/telemetry-utils';
+import { IFluidContainer } from '@fluidframework/fluid-static';
+import { IMember } from '@fluidframework/fluid-static';
+import { IServiceAudience } from '@fluidframework/fluid-static';
+import { ITelemetryBaseLogger } from '@fluidframework/common-definitions';
+import { ITokenProvider } from '@fluidframework/azure-client';
+import { ServiceAudience } from '@fluidframework/fluid-static';
+
+// @alpha (undocumented)
```

Comment:
I know this is still a draft, but ideally any of the items currently marked as `undocumented` should be documented before we merge. More details on these items may also help reviewers evaluate the API surface.